### PR TITLE
Data-Driven Keyboard Conversions: H, Part 2

### PR DESCRIPTION
**Pull request by noroadsleft**
## Description

Converts `rules.mk` entries to data-driven where applicable. Renames `info.json` to `keyboard.json` in instances where `rules.mk` can be safely deleted.

- keyboards with names matching `handwired/[0-9a-m]*/`

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/brain/keyboard.json [hunk 1]
```diff
@@ -46,6 +46,11 @@
     },
     "processor": "atmega32u4",
     "bootloader": "atmel-dfu",
+    "features": {
+        "bootmagic": true,
+        "mousekey": false,
+        "extrakey": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/brain/keyboard.json:53]:
```suggestion
```

File: keyboards/handwired/chiron/keyboard.json [hunk 2]
```diff
@@ -39,6 +39,13 @@
     },
     "processor": "atmega32u4",
     "bootloader": "atmel-dfu",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": false,
+        "rgblight": true,
+        "sleep_led": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/chiron/keyboard.json:48]:
```suggestion
```

File: keyboards/handwired/d48/rules.mk [hunk 3]
```diff
@@ -1,18 +1,3 @@
...
-#
-BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = no    # Mouse keys
-EXTRAKEY_ENABLE = yes   # Audio control and System control
-CONSOLE_ENABLE = yes    # Console for debug
-COMMAND_ENABLE = no     # Commands for debug and configuration
-NKRO_ENABLE = yes           # Enable N-Key Rollover
-BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
-AUDIO_ENABLE = yes
 USE_I2C = no
```
Comment (fauxpark) [inline keyboards/handwired/d48/rules.mk:1]:
```suggestion
```
This doesn't do anything (also found this in `naked64` and `setta21`).

File: keyboards/handwired/dactyl_manuform/4x5/keyboard.json [hunk 4]
```diff
@@ -33,6 +33,12 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/4x5/keyboard.json:41]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/4x5_5/keyboard.json [hunk 5]
```diff
@@ -26,6 +26,13 @@
         "soft_serial_pin": "D0"
     },
     "development_board": "promicro",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true,
+        "nkro": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/4x5_5/keyboard.json:35]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/4x6/keyboard.json [hunk 6]
```diff
@@ -33,6 +33,12 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/4x6/keyboard.json:41]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/4x6_5/keyboard.json [hunk 7]
```diff
@@ -33,6 +33,12 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/4x6_5/keyboard.json:41]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/5x6/keyboard.json [hunk 8]
```diff
@@ -33,6 +33,12 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6/keyboard.json:41]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/5x6_2_5/keyboard.json [hunk 9]
```diff
@@ -30,6 +30,11 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6_2_5/keyboard.json:37]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/5x6_5/keyboard.json [hunk 10]
```diff
@@ -30,6 +30,12 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "nkro": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6_5/keyboard.json:38]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/5x6_6/keyboard.json [hunk 11]
```diff
@@ -27,6 +27,13 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true,
+        "nkro": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x6_6/keyboard.json:36]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/5x7/keyboard.json [hunk 12]
```diff
@@ -33,6 +33,12 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/5x7/keyboard.json:41]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/6x6_4/keyboard.json [hunk 13]
```diff
@@ -33,6 +33,12 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_manuform/6x6_4/keyboard.json:41]:
```suggestion
```

File: keyboards/handwired/dactyl_promicro/keyboard.json [hunk 14]
```diff
@@ -33,6 +33,12 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_promicro/keyboard.json:41]:
```suggestion
```

File: keyboards/handwired/dactyl_rah/keyboard.json [hunk 15]
```diff
@@ -27,6 +27,12 @@
   },
   "processor": "atmega32u4",
   "bootloader": "caterina",
+  "features": {
+    "bootmagic": false,
+    "mousekey": true,
+    "extrakey": true,
+    "command": true
+  },
```
Comment (fauxpark) [inline keyboards/handwired/dactyl_rah/keyboard.json:35]:
```suggestion
```

File: keyboards/handwired/elrgo_s/keyboard.json [hunk 16]
```diff
@@ -27,6 +27,11 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/elrgo_s/keyboard.json:34]:
```suggestion
```

File: keyboards/handwired/freoduo/keyboard.json [hunk 17]
```diff
@@ -53,6 +53,14 @@
...
 "processor": "atmega32u4",
 "bootloader": "caterina",
+"features": {
+    "bootmagic": false,
+    "mousekey": true,
+    "extrakey": true,
+    "nkro": true,
+    "rgblight": true,
+    "velocikey": true
+},
```
Comment (fauxpark) [inline keyboards/handwired/freoduo/keyboard.json:63]:
```suggestion
```

File: keyboards/handwired/jtallbean/split_65/keyboard.json [hunk 18]
```diff
@@ -33,6 +33,13 @@
     },
     "processor": "atmega32u4",
     "bootloader": "atmel-dfu",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "console": true,
+        "command": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/jtallbean/split_65/keyboard.json:42]:
```suggestion
```

File: keyboards/handwired/ks63/keyboard.json [hunk 19]
```diff
@@ -27,6 +27,12 @@
     },
     "processor": "atmega32u4",
     "bootloader": "caterina",
+    "features": {
+        "bootmagic": false,
+        "mousekey": true,
+        "extrakey": true,
+        "command": true
+    },
```
Comment (fauxpark) [inline keyboards/handwired/ks63/keyboard.json:35]:
```suggestion
```

File: keyboards/handwired/lagrange/rules.mk [hunk 20]
```diff
@@ -1,17 +1,3 @@
...
-MOUSEKEY_ENABLE = no        # Mouse keys
-EXTRAKEY_ENABLE = yes       # Audio control and System control
-CONSOLE_ENABLE = yes        # Console for debug
-COMMAND_ENABLE = no         # Commands for debug and configuration
-NKRO_ENABLE = no            # Enable N-Key Rollover
-BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
-AUDIO_ENABLE = no           # Audio output
-UNICODE_ENABLE = yes
 SPLIT_TRANSPORT = custom
```
Comment (fauxpark) [inline keyboards/handwired/lagrange/rules.mk:1]: Can move this to `split.transport.protocol`.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**